### ISSUE TITLE
Afficher toute une rue

### ISSUE THREAD

**Issue by XioNoX** [cartesapp/cartes#318]
Chercher "Rue de Paris Brest" renvoi plusieurs suggestions, la première est:

https://cartes.app/?allez=Rue+de+Paris%7Cw304172919%7C-4.4665%7C48.4030#21.6/48.4029428/-4.4666767

Visualisé comme un point au centre d'un petit segment de la rue (car découpé sur OSM). Il serait utile de mettre en avant tout le linéaire de la rue.

**Comment by waterced** [comment]
Oui, d'autant plus que ces petits segments sont dans une relation de type `associatedStreet`

**Comment by etienneJr** [comment]
pris en compte dans #948 pour le cas d'un way dans une `associatedStreet`
